packages/angular/src/builders/webpack-browser/webpack-browser.impl.ts

```diff
@@ -69,12 +88,41 @@ function run(
     dependencies
   );
 
+  // If we don't have a third-party builder being used
+  // And there is a path to custom webpack config
+  // Invoke our own support for custom webpack config
+  let runBuildWithCustomWebpackConfig = { shouldRun: false, pathToConfig: '' };
+  if (
+    !options.targetBuilder &&
+    options.customWebpackConfig &&
+    options.customWebpackConfig.path
+  ) {
+    const pathToWebpackConfig = normalize(
+      join(context.workspaceRoot, options.customWebpackConfig.path)
+    );
+
+    if (existsSync(pathToWebpackConfig)) {
+      runBuildWithCustomWebpackConfig = {
+        shouldRun: true,
+        pathToConfig: pathToWebpackConfig,
+      };
+    }
```

Comment:
Should we throw an error if the configuration is not correct? I think having specified the `customWebpackConfig` with no `path` or an invalid one should result in a bad configuration error. What do you think?

Reply:
I had originally thought that if someone had provided the custom config incorrectly we wouldn't fail them over it, but perhaps if they're manually editing the `angular.json` or the `workspace.json` they would rather be told that they've made a mistake.

Yes I think we should throw some kind of error.

Reply:
Yes, it's better to be explicit so the devs know what's going on. If we silently fall back to not use a custom configuration, folks won't know why that happened.